tests/_support/Page/Acceptance/Administrator/ControlPanelPage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator\AdminPage;
+
+class ControlPanelPage extends AdminPage
+{
+    public static $url = "/administrator/index.php";
+
+    public static $pageTitle = 'Control Panel';
+
+    public static $pageTitleContext = ['class' => 'page-title'];
```

Comment:
I like the locator method idea, but then I don't know how we will be able to do this: https://www.youtube.com/watch?v=dV7fVtRz1SE